### DISCUSSION TITLE
5. 네트워크 - 물리 계층

### DISCUSSION THREAD

**Discussion by seongeun42** [Shortudy-10th/tech-for-developer#159] [질의 응답]
물리 계층 질의응답

**Comment by seongeun42** [comment]
## Q. 물리 계층의 역할은 무엇인가요?

물리 계층은 데이터를 전송하기 위해 허브, 케이블 같은 물리 장치를 사용하여 전기적 신호를 비트로 변환해 전송하는 역할을 합니다.

**Comment by seongeun42** [comment]
## Q. 물리 계층의 전송 단위는 무엇인가요?

비트입니다.

**Comment by zini9188** [comment]
## Q. 허브의 역할이 무엇인가요?

전기 신호를 증폭시켜 PC끼리 통신이 가능하도록 해주는 역할을 합니다.

**Comment by LireEruel** [comment]
## Q. 물리계층에서 디지털에서 아날로그로 변환하는 방식 2가지를 얘기해보세요.

물리계층에서 디지털에서 아날로그로 변환하는 방식은 2가지가 있습니다.
첫번쨰로 ASK (Amplitude Shift Keying, 진폭 변조)입니다.
ASK는 디지털 비트 0과 1을 아날로그 신호의 진폭으로 변조하는 방식입니다.
디지털 비트 0은 아날로그 신호의 진폭을 낮게 설정하고, 디지털 비트 1은 아날로그 신호의 진폭을 높게 설정하여 변조합니다.
두번째로 PSK (Phase Shift Keying, 위상 변조)입니다.
PSK는 디지털 비트 0과 1을 아날로그 신호의 위상(신호의 시작 시점)으로 변조하는 방식입니다.
디지털 비트 0은 아날로그 신호의 위상을 변화시키지 않고, 디지털 비트 1은 위상을 변경하여 변조합니다.

**Comment by watty-exclamationMark** [comment]
## 리피터란 무엇인가요?
허브와 마찬가지로 데이터 신호를 증폭하여 전송 거리를 확장하는 역할을 합니다.

**Reply by watty-exclamationMark** [reply]
## 리피터와 허브의 차이점
사실 허브는 리피터에 몇가지 기능을 추가한 것입니다.

허브는 리피터와 다르게 패킷 모니터링과 멀티 포트를 지원하여 트래픽 관리가 가능합니다.
